execution/gethexec/switch_sequencer.go

```diff
@@ -0,0 +1,148 @@
+package gethexec
```

Comment:
This module allows the nitro sequencer running on Espresso to automatically bypass the Espresso sequencer in case of a liveness failure to avoid stalling Nitro.